src/picongpu/include/plugins/PhaseSpace/PhaseSpaceFunctors.hpp

```diff
@@ -107,8 +107,8 @@ namespace picongpu
              * p_bin < 0 ? p_bin = 0;
              * p_bin > (num_pbins-1) ? p_bin = num_pbins-1;
              */
-            p_bin *= int(p_bin >= 0);
-            p_bin += int(p_bin >= num_pbins) * (num_pbins - 1 - p_bin);
+            p_bin *= p_bin >= 0 ? 1 : 0;
+            p_bin += p_bin >= num_pbins ? num_pbins-1-p_bin : 0;
```

Comment:
Thanks. I did not know that this is valid `C++/C`. 
With the format suggested, I need to check, if branch occures or if the `nvcc` avoids it.